app/2.0.x/index.md

```diff
@@ -5,19 +5,19 @@ title: Documentation for Kong
 <div class="docs-grid">
   <div class="docs-grid-block">
     <h3><img src="/assets/images/icons/documentation/icn-window.svg" /><a href="https://konghq.com/install/">Installation</a></h3>
-    <p>You can install Kong on most Linux distributions and macOS. We even provide the source so you can compile yourself.</p>
+    <p>You can install Kong on most Linux distributions and MacOS. We even provide the source so you can compile it yourself.</p>
     <a href="https://konghq.com/install/">Install Kong &rarr;</a>
   </div>
 
   <div class="docs-grid-block">
     <h3><img src="/assets/images/icons/documentation/icn-quickstart.svg" /><a href="/{{page.kong_version}}/getting-started/quickstart">5-minute Quickstart</a></h3>
-    <p>Learn how to start Kong, add a Service, enable plugins, and add consumers in under thirty seconds.</p>
+    <p>Learn how to start Kong, add a Service, Enable plugins, and add consumers in under thirty seconds.</p>
```

Comment:
This doesn't need to be capitalized - Service is only caps because it's a Kong object, whereas "enable" is a verb.
```suggestion
    <p>Learn how to start Kong, add a Service, enable plugins, and add consumers in under thirty seconds.</p>
```

Reply:
Okay